### PR TITLE
Initial storybook integration

### PR DESCRIPTION
**Pull request by puehringer**
Adds `storybook` and `build-storybook` npm scripts to run and build the storybook application. 

image

### CODE REVIEW

File: src/vis/Vis.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    },
+    {
+      info: {
+        description: '',
+        id: 'category',
+        name: 'category',
+      },
+      type: EColumnTypes.CATEGORICAL,
+      values: () => dataPromise.then((data) => data.category.map((val, i) => ({ id: i.toString(), val }))),
+    },
```
Comment (thinkh) [inline src/vis/Vis.stories.tsx:66]: Should we add some proper names and description?

**Review by thinkh — COMMENTED** [review]
I have added some comments that can be either addressed in this PR or in the follow-up issue https://github.com/datavisyn/reprovisyn/issues/233.

File: src/vis/general/InvalidCols.tsx [hunk 2]
```diff
@@ -1,13 +1,9 @@
...
-interface InvalidColsProps {
-  message: string;
-}
-
-export function InvalidCols({ message }: InvalidColsProps) {
+export function InvalidCols({ message }: { message: string }) {
   return (
     <div className="card w-25 h-10 justify-content-center">
-      <div className="card-header">Invalid Columns Selected</div>
+      <div className="card-header">Invalid columns selected</div>
```
[the remark this answers is not in the document]
Comment (puehringer) [reply src/vis/general/InvalidCols.tsx:6]: @dvzacharycutler Can you extract that to the .json, the same way you extract the actual error message?

File: .storybook/preview.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import '../src/scss/storybook.scss';
+import { initializeLibrary } from '../src/initialize';
+
+// TODO: This is async, how to wait for it?
+initializeLibrary();
```
[the remark this answers is not in the document]
Comment (puehringer) [reply .storybook/preview.js:5]: It _works_ fine now and it never not worked, I just know that it is not _perfect_, such that I would keep it as is. We might want to look deeper into async stories when @dvzacharycutler is taking a closer look at storybook.

File: src/vis/Vis.stories.tsx (hunk 1 above)
Comment (puehringer) [reply src/vis/Vis.stories.tsx:66]: I just needed some random demo data to show something, but maybe we should show some data that makes sense? I am thinking about something like this [iris ](https://www.google.com/search?q=iris+dataset&oq=iris+dataset&aqs=chrome.0.0i512l10.1749j0j7&sourceid=chrome&ie=UTF-8)dataset? Do you have any dataset in mind where we can visually present all use-cases with a single dataset @thinkh  @dvzacharycutler?

**Pull request merged**